### PR TITLE
User can choose from menu

### PR DESCRIPTION
**Pull request by alexkim-is**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+      <a-entity id="links" layout="type: line; margin: 2" position="-1 2 -5">
+        <a-entity id="movie-link" template="src: #link" data-src="#minion" data-thumb="#video-thumb"></a-entity>
+        <a-entity id="game-link" template="src: #link" data-src="#game" data-thumb="#game-thumb"></a-entity>
+      </a-entity>
+
+      <a-entity id="play360" geometry="primitive: sphere; radius: 5000;
+                    segmentsWidth: 64; segmentsHeight: 64;"
+                    material="shader: flat; src: url(yosemite.mp4); stereoscopicType:RL"
+                    scale="-1 1 1" autoplay="false></a-entity>
```
Comment (thebearingedge) [inline index.html]: Looks like you have a syntax error here in `autoplay="false"`. It's missing a closing quote.

**Review by thebearingedge — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (alexkim-is) [reply index.html]: Thanks Tim!

Comment (alexkim-is) [reply index.html]: Fixed! along w/ others changes

**Comment by thebearingedge** [comment]
Closes #1

**Comment by thebearingedge** [comment]
Closes #1

**Pull request merged**